wagtail/admin/widgets/datetime.py

```diff
@@ -119,13 +121,15 @@ def __init__(self, attrs=None, format=None, time_format=None):
             time_fmt = getattr(settings, "WAGTAIL_TIME_FORMAT", DEFAULT_TIME_FORMAT)
         self.js_format = to_datetimepicker_format(fmt)
         self.js_time_format = to_datetimepicker_format(time_fmt)
+        self.js_parent_selector = parent_selector
         super().__init__(attrs=default_attrs, format=fmt)
 
     def get_config(self):
         return {
             "dayOfWeekStart": get_format("FIRST_DAY_OF_WEEK"),
             "format": self.js_format,
             "formatTime": self.js_time_format,
+            "parentID": self.js_parent_selector,
```

Comment:
Just to double check – you called this `parent_selector`, because despite the jQuery UI widget calling its attribute `parentID`, it can take an arbitrary selector? If so – makes sense to me, but could be worth a comment to clarify.

Reply:
Yep, correct.

Reply:
I've added a comment to this line.